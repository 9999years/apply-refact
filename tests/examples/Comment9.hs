-- input
foo = f <$> {- comment attached to x -} x >>= g
