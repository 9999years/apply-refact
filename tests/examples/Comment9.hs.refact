[("tests/examples/Comment9.hs:2:7-47: Warning: Redundant <$>\nFound:\n  f <$> x >>= g\nPerhaps:\n  x >>= g . f\n",[Replace {rtype = Expr, pos = SrcSpan {startLine = 2, startCol = 7, endLine = 2, endCol = 48}, subts = [("f",SrcSpan {startLine = 2, startCol = 7, endLine = 2, endCol = 8}),("g",SrcSpan {startLine = 2, startCol = 47, endLine = 2, endCol = 48}),("x",SrcSpan {startLine = 2, startCol = 41, endLine = 2, endCol = 42})], orig = "x >>= g . f"}])]
