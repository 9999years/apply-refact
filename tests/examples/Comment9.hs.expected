-- input
foo =  {- comment attached to x -}x >>= g . f
